iib/web/api_v1.py

```diff
@@ -90,6 +91,23 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
```

Comment:
Can we please also add `:return: json object representing the health of IIB.`

Reply:
Added.